Gitpod build error

### What browser are you using?

Other

### Other browser name

Problem is not with browser

### Describe the bug

The Gitpod container doesn't work for me anymore. The error message is at the bottom of this console output:

```bash
gitpod /workspace/renovatebot.github.io (main) $  HISTFILE=/workspace/.gitpod/cmd-0 history -r; {
> nvm install 16 && nvm use 16 && make
> 
> } && {
> make serve
> }
v16.17.1 is already installed.
Now using node v16.17.1 (npm v8.15.0)
Now using node v16.17.1 (npm v8.15.0)
yarn install --frozen-lockfile
yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
[4/5] Linking dependencies...
[5/5] Building fresh packages...
$ husky install
husky - Git hooks installed
Done in 22.88s.
pipenv install
Creating a virtualenv for this project...
Pipfile: /workspace/renovatebot.github.io/Pipfile
Using /home/gitpod/.pyenv/versions/3.8.13/bin/python (3.8.13) to create virtualenv...
⠧ Creating virtual environment...created virtual environment CPython3.8.13.final.0-64 in 895ms
  creator CPython3Posix(dest=/workspace/renovatebot.github.io/.venv, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/gitpod/.local/share/virtualenv)
    added seed packages: pip==22.2.2, setuptools==65.3.0, wheel==0.37.1
  activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator

✔ Successfully created virtual environment! 
Virtualenv location: /workspace/renovatebot.github.io/.venv
Installing dependencies from Pipfile.lock (04f315)...
To activate this project's virtualenv, run pipenv shell.
Alternatively, run a command inside the virtualenv with pipenv run.
bash bin/get-docs.sh
yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
warning Resolution field "clipanion@3.2.0-rc.12" is incompatible with requested version "clipanion@3.2.0-rc.4"
[3/5] Fetching packages...
[4/5] Linking dependencies...
warning " > clipanion@3.2.0-rc.12" has unmet peer dependency "typanion@*".
warning "@yarnpkg/core > @yarnpkg/shell > clipanion@3.2.0-rc.4" has unmet peer dependency "typanion@*".
warning Workspaces can only be enabled in private projects.
warning Workspaces can only be enabled in private projects.
warning Workspaces can only be enabled in private projects.
warning Workspaces can only be enabled in private projects.
warning Workspaces can only be enabled in private projects.
[5/5] Building fresh packages...
$ run-s prepare:*
yarn run v1.22.19
$ husky install
husky - Git hooks installed
Done in 0.15s.
yarn run v1.22.19
$ run-s generate:*
$ node tools/generate-imports.mjs
generating imports
> data/azure-pipelines-tasks.json
> data/debian-distro-info.json
> data/kubernetes-api.json5
> data/node-js-schedule.json
> data/ubuntu-distro-info.json
> node_modules/emojibase-data/en/shortcodes/github.json
generating hashes
Done in 0.98s.
yarn run v1.22.19
$ npm explore re2 -- npm run install

> re2@1.17.7 install
> install-from-cache --artifact build/Release/re2.node --host-var RE2_DOWNLOAD_MIRROR --skip-path-var RE2_DOWNLOAD_SKIP_PATH --skip-ver-var RE2_DOWNLOAD_SKIP_VER || npm run rebuild

Trying https://github.com/uhop/node-re2/releases/download/1.17.7/linux-x64-93.br ...
Writing to build/Release/re2.node ...
Done.
Done in 2.00s.
yarn run v1.22.19
$ node tools/check-re2.mjs
Checking re2 ... ok.
Done in 0.21s.
Done in 36.72s.
yarn run v1.22.19
$ run-s "release:prepare {@}" --
$ node -r ts-node/register/transpile-only -- tools/generate-docs.ts
 INFO: generating docs
 INFO: * static
 INFO: * platforms
 INFO: * versionigs
 INFO: * datasources
 INFO: * managers
 INFO: * presets
 INFO: * templates
 INFO: * configuration-options
 INFO: * self-hosted-configuration
 INFO: * json-schema
Done in 15.66s.
pipenv run mkdocs build
Traceback (most recent call last):
  File "/workspace/renovatebot.github.io/.venv/bin/mkdocs", line 5, in <module>
    from mkdocs.__main__ import cli
  File "/workspace/renovatebot.github.io/.venv/lib/python3.8/site-packages/mkdocs/__main__.py", line 15, in <module>
    from mkdocs import __version__, config, utils
  File "/workspace/renovatebot.github.io/.venv/lib/python3.8/site-packages/mkdocs/config/__init__.py", line 1, in <module>
    from mkdocs.config.base import Config, load_config
  File "/workspace/renovatebot.github.io/.venv/lib/python3.8/site-packages/mkdocs/config/base.py", line 25, in <module>
    from mkdocs import exceptions, utils
  File "/workspace/renovatebot.github.io/.venv/lib/python3.8/site-packages/mkdocs/utils/__init__.py", line 37, in <module>
    from importlib_metadata import EntryPoint, entry_points
ModuleNotFoundError: No module named 'importlib_metadata'
make: *** [Makefile:15: build-docs] Error 1
```

### Steps to reproduce

1. Open the `main` branch in Gitpod. [^gitpod-link]
1. Let Gitpod run these steps: https://github.com/renovatebot/renovatebot.github.io/blob/714aeeeff4d682c38681b26cf0ccb67be1d0b35c/.gitpod.yml#L1-L11

1. Get error message posted above.

[^gitpod-link]: https://gitpod.io/#https://github.com/renovatebot/renovatebot.github.io/tree/main

### Additional context

Looks like Gitpod can't find the `'importlib_metadata'` module anymore:

```bash
ModuleNotFoundError: No module named 'importlib_metadata'
```

After starting a new Gitpod workspace on the latest `main` branch, I get this error:

```
gitpod /workspace/renovatebot.github.io (main) $  HISTFILE=/workspace/.gitpod/cmd-0 history -r; {
> nvm install 16 && nvm use 16 && make
> 
> } && {
> make serve
> }
v16.18.0 is already installed.
Now using node v16.18.0 (npm v8.19.2)
Now using node v16.18.0 (npm v8.19.2)
yarn install --frozen-lockfile
yarn install v1.22.19
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
[4/5] Linking dependencies...
[5/5] Building fresh packages...
$ husky install
husky - Git hooks installed
Done in 25.71s.
pipenv install
pyenv: version `3.10.7' is not installed (set by /workspace/renovatebot.github.io/.python-version)
make: *** [Makefile:5: install] Error 1
```

It looks like just having the new `.python-version` file is not enough for Gitpod to understand what we want. Maybe we need to change the `gitpod.yml` file, or create our own Docker image that has the correct versions of everything?

https://github.com/renovatebot/renovatebot.github.io/blob/c65f8fabb98067ab66f37d367734bbe3d955be45/.gitpod.yml#L1-L11

--- a/.gitpod.yml
+++ b/.gitpod.yml
@@ -2,6 +2,7 @@ tasks:
   - init: >
       nvm install 16 &&
       nvm use 16 &&
+      pyenv install &&
       pip3 install pipenv &&
       make
     command: make serve

